### PR TITLE
Remove the “…” menu in the sidebar

### PR DESCRIPTION
**Pull request by j-f1**
I was introducing a relative to Actual, and they were confused by the existence of two “…” menu buttons. Now there’s just one!

UI:
- Find Schedules is now a button on the Schedules page
- the Settings and Help options have been moved to the More Tools panel
  - clarify that the settings apply only to the current budget
  - maybe rephrase “Actual Help”? I just realized it sounds like click here for the _actual_ help page, not some impostor
- Fix Split Transactions has been removed since the issues it fixes were never present in the OSS version as far as I can tell
- Move the Close File action to a “back” button to the left of the budget name
- remove the now-empty “…” menu at the top left
- since the “<<” icon is now confusable with the back button, I changed it to be a pin (which also matches its behavior)
- add a “Help” link to the bottom left corner of the management app

Screenshot_2023-01-02 12 20 05

Code:
- refactor the Settings page into ~1 file for each section
- The code that displays notifications has been commented out since all of the notifications are from before the codebase was OSS and are therefore no longer relevant. Not deleting the code so it can serve as a template for future notifications

### CODE REVIEW

**Comment by YagiOoda** [comment]
Instead of "Actual Help", maybe "Actual Docs"? Alternatively, "Actual" could be taken out and it could be relabeled to "Help Pages" or something similar.

**Comment by j-f1** [comment]
maybe “Documentation?”

**Comment by MatissJanis** [comment]
👋 There's quite a lot of changes in this PR. Would you mind breaking them up into smaller, isolated PRs?

**Comment by jlongster** [comment]
Personally, I think this is a worse UX overall. The back arrow in the very top left, a prime real estate in the app, makes it feel like you are in a modal state and introduces some anxiety (when do I need to back? why would I go back?) Also, the action is closing the file, and the back arrow makes it look like a backwards navigation. Closing a file is done very infrequently and should have some a large presence in the UI. 

It's very common to have a "..." menu that provides more actions in the app. I personally think the problem is more with the "more tools" item in the menu and we should find a different solution for that instead of getting rid of "...". One option would be go back to previous iteration where the items are nested under "more tools", and "more tools" should be renamed to something like "management" (dunno exactly...). I know I previously suggested avoiding the nested menu but maybe that was a mistake.

Another option is to come up with a name for a the item in the sidebar (management? items? data?) and that takes you to a page with a secondary navigation at the top. The horizontal navigation would have stuff like "payees", "rules", etc.

File: packages/desktop-client/src/components/accounts/Account.js [hunk 1]
```diff
@@ -567,13 +567,13 @@ function SelectedTransactionsButton({
...
             if (scheduleId) {
-              history.push(`/schedule/edit/${scheduleId}`, {
+              history.push(`/schedules/edit/${scheduleId}`, {
                 locationPtr: history.location
               });
             }
             break;
           case 'link-schedule':
-            history.push(`/schedule/link`, {
+            history.push(`/schedules/link`, {
```
Comment (jlongster) [inline packages/desktop-client/src/components/accounts/Account.js:576]: I'd avoiding changing URLs unless you have a strong reason to. This makes it inconsistent with others and the singular makes more sense because you are linking an individual schedule.

Comment (j-f1) [reply packages/desktop-client/src/components/accounts/Account.js:576]: Good point! I didn’t see that other things used the singular so it makes sense to stick with the previous scheme here. I made this change because I noticed that the “schedules” button in the sidebar was getting un-highlighted when clicking on the button to edit the schedule. I think that may be a deeper issue though where the link is styling itself based on the URL of the frontmost modal, not the actual page that’s open.

Comment (j-f1) [reply packages/desktop-client/src/components/accounts/Account.js:576]: (maybe this is a reason to change up how that system works though, I’ve noticed that reloading while a modal is open crashes the app until you change the URL)

**Comment by j-f1** [comment]
While I see your concerns about the back button being confusing, I think there’s definitely some precedent for a button that works this way. For example, in the Google Docs web apps, clicking the document/logo icon in the top left corner takes you back “home“ to a list of recently edited documents. On iPhones, both the Google Docs apps and the iWork apps (like pages) use a back button to suggest “close.” Figma has it as the first option under a menu at the top left.

While this pattern is probably more common on mobile than on desktop, I think it makes sense here because Actual is not a desktop app so we don’t get the benefit of window buttons providing an obvious affordance to close the document, and we want to keep the entire area of the screen usable for content rather than having a global navigation bar along one of the edges like other apps have.

That said, there is definitely a better way than my proposal and I’m happy to put this on pause until someone comes up with something. Will split out the other parts of this PR separately though!

File: packages/desktop-client/src/components/accounts/Account.js (hunk 1 above)
Comment (jlongster) [reply packages/desktop-client/src/components/accounts/Account.js]: Oh I see, good point. Yeah there's some weirdness there! Overall if you land some stuff separately like Matiss suggested it will be easier to review :)

**Comment by MatissJanis** [comment]
@j-f1 should this PR be closed? Anything else you're planing to do here?

**Comment by j-f1** [comment]
I still think there are improvements in here, but I’m not terribly enthusiastic about handling more design discussions about the sidebar, so I’m gonna close this for now.
